[13.0][MIG] account_document_reversal, migration to 13.0

kittiu opened the pull request:
Migration with some conceptual changes (because no more account.inovice).
Please read on the README.

Ready for review.

kittiu committed:
[12.0][ADD] account_document_reversal

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

oca-travis committed:
[UPD] Update account_document_reversal.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

alvarorib committed:
Added translation using Weblate (Portuguese)

alvarorib committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (34 of 34 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_document_reversal
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_document_reversal/pt/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 47.1% (16 of 34 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_document_reversal
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_document_reversal/hr/

AaronHForgeFlow commented:
It seems the module https://github.com/OCA/account-financial-tools/tree/13.0/account_document_reversal is already available in v13 :astonished:

kittiu commented:
> It seems the module https://github.com/OCA/account-financial-tools/tree/13.0/account_document_reversal is already available in v13

Nop, just today that we are having this problem. The one I accidentally merged 12 days ago, just seem to be auto merged (so, what is merged now was the unfinished product).
We are just discussing it, https://github.com/OCA/account-financial-tools/pull/956

kittiu commented:
@AaronHForgeFlow back to green now :)

kittiu commented:
@Saran440 please help review if you have time. Thanks!

oca-travis committed:
[UPD] Update account_document_reversal.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_document_reversal
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_document_reversal/

fernandahf committed:
[FIX] account_document_reversal: modify to correct development status

OCA-git-bot committed:
account_document_reversal 12.0.1.0.1

Saran440 commented:
Functional Test :+1: 
if you check cancel reversal in Payment menu should hide reset to draft.
 
![Selection_001](https://user-images.githubusercontent.com/20896369/99934137-ba0e3a80-2d8f-11eb-83e3-b2aa94bb12f7.png)

hhgabelgaard committed:
Added translation using Weblate (Danish)

hhgabelgaard committed:
Translated using Weblate (Danish)

Currently translated at 88.2% (30 of 34 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_document_reversal
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_document_reversal/da/

JordiBForgeFlow commented on account_document_reversal/__manifest__.py:
```diff
@@ -0,0 +1,23 @@
+    "name": "Account Document Reversal",
+    "summary": "Create reversed journal entries when cancel document",
+    "version": "13.0.1.0.0",
+    "author": "Ecosoft," "Eficent," "Odoo Community Association (OCA)",
```
@kittiu can you change Eficent -> ForgeFlow?

kittiu replied:
Done

JordiBForgeFlow commented:
/ocabot merge nobump

kittiu committed:
[IMP] account_document_reversal black, isort, prettier

kittiu committed:
[13.0][MIG] account_document_reversal

JordiBForgeFlow commented:
cannot merge due to fail in pre-commit https://github.com/OCA/account-financial-tools/runs/4773419136?check_suite_focus=true#step:4:138. Development_status = "Beta" not allowed?¿ @pedrobaeza Can you help?

pedrobaeza commented:
Not sure, but I would remove the development_status key, as it's the default one.

JordiBForgeFlow commented:
@kittiu if you proceed to remove the development_status I can try to merge again.

kittiu commented:
> @kittiu if you proceed to remove the development_status I can try to merge again.

Removed, shall you try again please.

AaronHForgeFlow commented:
/ocabot merge nobump
